libmamba/include/mamba/core/graph_util.hpp

```diff
@@ -340,6 +344,12 @@ namespace mamba
         return m_node_list.size();
     }
 
+    template <typename N, typename G>
+    inline auto DiGraphBase<N, G>::number_of_edges() const -> std::size_t
```

Comment:
`inline` is redundant here ^^

Reply:
I noticed this, yes. Perhaps it is better to do it in another PR since this file is not widely modified by here.

Reply:
The rest is done there https://github.com/mamba-org/mamba/pull/2039